### DISCUSSION TITLE
Trivy flagging withdrawn github security advisory (https://github.com/advisories/GHSA-8x6c-cv3v-vp6g)

### DISCUSSION THREAD

**Discussion by binsonc** [aquasecurity/trivy#3611] [Q&A] [answered]
Hello all,

We are using trivy 0.37.2, it is currently flagging this advisory (https://github.com/advisories/GHSA-8x6c-cv3v-vp6g) as a vulnerability, even though the advisory has been withdrawn.

Does trivy take some time to reflect this change or is it an issue with the GHSA itself?

**Comment by knqyf263** [comment, marked as answer]
The DB is updated every 6 hours. Please try scanning later.
